main/nav_backplane_viewer.py

```
@@ -0,0 +1,1366 @@
+        group = self._image_groups[index]
+        if len(group.image_files) != 1:
+            self._logger.error('Expected single image per group; got %d', len(group.image_files))
+            return
+        image_file = group.image_files[0]
+        image_path = image_file.image_file_path.absolute()
+        results_stub = image_file.results_path_stub
+        try:
+            self._current_image_name = image_path.name
+            self.setWindowTitle(f'Backplane Viewer - {self._current_image_name}')
+        except Exception:
+            pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)